src/pynxtools/nomad/schema.py

```diff
@@ -913,6 +947,29 @@ def normalize_sample(self, archive, logger):
     super(current_cls, self).normalize(archive, logger)
 
 
+def normalize_entry(self, archive, logger):
+    """Normalizer for sample section."""
```

Comment:
These docstrings (also the one below) are not correct

Reply:
fixed